libvmi/accessors.c

```diff
@@ -335,10 +335,10 @@ GSList* vmi_get_va_pages(vmi_instance_t vmi, addr_t dtb) {
 
 addr_t vmi_pagetable_lookup (vmi_instance_t vmi, addr_t dtb, addr_t vaddr)
```

Comment:
...and the code shouldn't be duped in any case, so making this just a wrapper to `vmi_pagetable_lookup_cache()` if we decide not to change the external API.